KEP-2086: update docs for ServiceInternalTrafficPolicy (#37912)
kubernetes/website: content/en/docs/concepts/services-networking/service-traffic-policy.md

<!-- overview -->

> {{< feature-state for_k8s_version="v1.26" state="GA" >}}

Reviewer: stable:
`ERROR 2022/11/15 15:45:38 "GA" is not a valid feature-state, use one of "alpha, beta, deprecated, stable"`

Author: Done, thanks

## Using Service Internal Traffic Policy

> The `ServiceInternalTrafficPolicy` feature (GA in v1.26) enables the internal-only
> traffic policy for a {{< glossary_tooltip text="Service" term_id="service" >}}, by setting its
> `.spec.internalTrafficPolicy` to `Local`. This tells kube-proxy to only use node local
> endpoints for cluster internal traffic.

Reviewer 2: ```suggestion
You can enable the internal-only
traffic policy for a {{< glossary_tooltip text="Service" term_id="service" >}}, by setting its
`.spec.internalTrafficPolicy` to `Local`. This tells kube-proxy to only use node local
endpoints for cluster internal traffic.
```

We document stable features as if the text is being read several releases hence. See https://developers.google.com/style/timeless-documentation for some similar thinking.

The feature state shortcode is there to serve as a big clue that previous versions might not have included it.

Author: Thanks, used your suggestion

`spec.internalTrafficPolicy` setting. When it's set to `Local`, only node local

> endpoints are considered. When it's `Cluster` or missing, all endpoints are
> considered.
> `spec.internalTrafficPolicy` defaults to "Cluster".

Reviewer 2: ```suggestion
endpoints are considered. When it's set to `Cluster` (the default), or is not set, Kubernetes
considers all endpoints.
```

Author: Thanks, used this suggestion
